lomap/dbmol.py

```diff
@@ -125,21 +126,27 @@ def __init__(self, directory, parallel=1, verbose='off',
             if not isinstance(display,  bool):
                 raise TypeError('The display flag is not a bool type')
 
+            if not isinstance(radial,  bool):
+                raise TypeError('The display flag is not a bool type')
```

Comment:
Yes, you are right. It's my fault, will change to the "The radial flag is not a bool type"